staff-commands/clean.md

```diff
@@ -34,6 +34,7 @@ This command is used to delete multiple messages from the channel it's used in.
 
 ```
 m:clean (all|user|matching|notmatching|bots|system|startswith|endswith) [...data] <amount>
+/clean (all|user|matching|notmatching|bots|system|startswith|endswith) [...data] <amount>
```

Comment:
Please update this to reflect the actual new functioning of the command. (This is the command with most changes)